eng/service.proj

```diff
@@ -51,6 +51,14 @@
              SkipNonexistentTargets="true" />
   </Target>
 
+  <Target Name="GenerateTests">
+    <MSBuild Projects="@(ProjectReference)"
+             Targets="GenerateTest"
```

Comment:
Is there already a GenerateTest target in the library targets somewhere? A quick search didn't show anything.

Reply:
Yes @weshaggard, there already be a GenerateTest target. It searches "autorest.tests.md" in tests folder and generate test. The file is expected to be like this: https://github.com/Azure/azure-sdk-for-net/blob/2a79d3ba5592bdac5456d8b0fd7a8fa1e6433e1c/sdk/servicebus/Azure.ResourceManager.ServiceBus/tests/autorest.tests.md

For now, we didn't commit "autorest.tests.md" to any RPs yet, so it generates nothing currently. It's expected to be enabled after we finish scenario test generation development.